packages/terra-date-picker/src/react-datepicker/day.jsx

```diff
@@ -98,11 +98,18 @@ class Day extends React.Component {
   }
 
   handleMouseEnter = (event) => {
-    if (!this.isDisabled() && this.props.onMouseEnter) {
+    if (!this.isDisabled() && this.props.onMouseEnter && !this.props.inline) {
```

Comment:
Why not trigger onMouseEnter if it's an in-line calendar?

Reply:
Because onMouseEnter sets selectingDate state value which causes selected dotted border to be appeared on days while entering calendar filter component. ( prevents re-render which happens on OnMouseEnter )

Reply:
So this is preventing adding the dotted border?

Reply:
yes, To prevent dotted border on mouse hover.

Reply:
update : we need to this condition to prevent focus border to appear on mouse enter when calendar is inline.